.travis.yml

```diff
@@ -0,0 +1,14 @@
+sudo: false
+language: python
+python:
+  - 2.7
```

Comment:
there may be some projects that require 3.4 and, esp. if we're hoping to get community contributors, it might not hurt to ensure that all our tests work on both.  but again, if we can just get travis enabled for now, i can create an issue to go thru and fix those tests that aren't working right now with 3.4.  if it's largely print() that's the culprit, that should be easy to fix.
created:
 [travis] fix tests for python3.4 compatibility #45